spec/factories/protocol.rb

```diff
@@ -124,7 +130,7 @@
     end
 
     factory :protocol_without_validations, traits: [:without_validations]
-    factory :study_without_validations, traits: [:without_validations, :study]
+    factory :protocol_study_without_validations, traits: [:without_validations, :study]
```

Comment:
Should rename to study_without_validations to be consistent with project_without_validations